### PR TITLE
Themis 0.14

### PR DESCRIPTION
**Pull request by ilammy**
Getting ready for the next release. Let's see how many things I forgot to update this time.

`release-0.14-prep` branch is in the main repo, feel free to throw in changes there directly.

## Checklist

- [X] Change is covered by automated tests
- [X] Changelog is updated
- [X] Versions are updated
- [X] Provisional 0.14.0 tag points to this branch

Published documentation pending review:

- [x] `CHANGELOG.md`
- [x] `README.md`
- [x] `gothemis/README.md`
- [x] `src/wrappers/themis/jsthemis/README.md`
- [x] `src/wrappers/themis/python/README.md`
- [x] `src/wrappers/themis/rust/README.md`
- [x] `src/wrappers/themis/rust/libthemis-sys/README.md`
- [x] `src/wrappers/themis/wasm/README.md`

Packaging files that will be updated later:

- `PKGBUILD.MSYS2` – once we have a stable tag and source tarball
- `Package.swift` – once we have a binary release ready
- `docs/examples`, `tools` – once release is published

### CODE REVIEW

File: src/wrappers/themis/ruby/rbthemis.gemspec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
   s.name        = 'rbthemis'
-  s.version     = '0.13.0'
-  s.date        = '2020-07-08'
+  s.version     = '0.14.0'
+  s.date        = '2021-12-24'
```
Comment (shadinua) [inline src/wrappers/themis/ruby/rbthemis.gemspec:4]: Do we actually plan to release exactly on Christmas day? :)

Comment (ilammy) [reply src/wrappers/themis/ruby/rbthemis.gemspec:4]: > Do we actually plan to release exactly on Christmas day? :)

I had to put *something* there vOv

The closest other “nice” date is the solstice, 2021-12-21. Are we going to make it? :D

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,27 @@
...
 
+
+## [0.14.0](https://github.com/cossacklabs/themis/releases/tag/0.14.0), December 24th 2021
+
+**TL;DR:**
+
+- JavaThemis for Android has moved to Maven Central ([read more](#0.14-maven-central)).
+- JavaThemis for Java is available on Maven Central ([read more](#0.14-maven-central)).
+- TypeScript definitions and ES6 module are now available for WasmThemis ([read more](#0.14-typescript)).
+- Node.js v16 is now supported.
```
Comment (vixentael) [inline CHANGELOG.md:15]:
```suggestion
- Node.js v16 is now supported.
- Themis for iOS is available through SPM, Cocoapods and Carthage. Uh.
```

File: src/wrappers/themis/ruby/rbthemis.gemspec (hunk 1 above)
Comment (vixentael) [reply src/wrappers/themis/ruby/rbthemis.gemspec:4]: we def won't make it 2021-12-21

**Review by vixentael — APPROVED** [review]
Let's update Readme Availability section as well?

File: CHANGELOG.md (hunk 2 above)
Comment (ilammy) [reply CHANGELOG.md:15]: Damn, I forgot that SPM support was not a part of 0.13.0, but appeared along the way. Now I gotta review the changes in the patch versions...

**Comment by ilammy** [comment]
While rereading changelogs for 0.13, I have noticed that we have one more change to CocoaPods subspecs to make. Don't be too quick to merge this PR, I have one extra thing to add to `release/0.14` before this one.

**Comment by ilammy** [comment]
Okay, merged in the subspec change and moved the provisional 0.14.0 tag to the updated branch so that podspec validation should actually succeed. Waiting on CI...

**Comment by vixentael** [comment]
haha, we had tests for boringssl!

**Comment by ilammy** [comment]
> haha, we had tests for boringssl!

Not funny ☹️ I'll save the jokes for the next PR.

Stand by, I need to unfuck my Ruby installation first to get CocoaPods to install dependencies, to test BoringSSL removal.

**Review by Lagovas — APPROVED** [review]
lgtm

**Comment by ilammy** [comment]
And once again 🤞 Third time's the charm, right?

**Pull request merged**